setup.py

```diff
@@ -5,8 +5,14 @@
 
 # Get a list of scripts to install
 scripts = []
-for root, _, filenames in os.walk('scripts'):
+for root, dirnames, filenames in os.walk('scripts'):
```

Comment:
From the docs os.walk needs `topdown=True` for the modified directories to work in all cases.